### ISSUE TITLE
Regression between 1.1.33 and 1.1.34

### ISSUE THREAD

**Issue by Stexxen** [springdoc/springdoc-openapi#84]
The Following
```java
  @Operation(summary = "Get thing",
          responses = {
                  @ApiResponse(description = "Successful Operation", responseCode = "200",
                          content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))),
                  @ApiResponse(responseCode = "404", description = "Not found"),
                  @ApiResponse(responseCode = "401", description = "Authentication Failure")
          }
  )
```

generates this in 1.1.33
```

    200:
          description: Successful Operation
          content:
            application/json:
              schema:
                type: string
        404:
          description: Not found
          content: {}
        401:
          description: Authentication Failure
          content: {}
```

but this in 1.1.34
```
        200:
          description: Successful Operation
          content:
            application/json:
              schema:
                type: string
        404:
          description: Not found
          content:
            '*/*':
              schema:
                type: string
        401:
          description: Authentication Failure
          content:
            '*/*':
              schema:
                type: string
```

Which is rendered as 
![image](https://user-images.githubusercontent.com/4494301/64415505-5ebe3280-d08d-11e9-9054-076fa7c6703f.png)

I think it was caused by #80 but not 100%

**Comment by springdoc** [comment]
Hi, 

 Unable to reproduce the regression you are describing between 1.1.33 and 1.1.34.
 Can you  please add, your controller details (method and parameters) and more elements in order to reproduce this issue?

**Comment by Stexxen** [comment]
Sure. Here is the complete controller
For reference I'm using Spring Boot 2.1.7.RELEASE
```
package xxx.poc;

import io.swagger.v3.oas.annotations.Operation;
import io.swagger.v3.oas.annotations.media.Content;
import io.swagger.v3.oas.annotations.media.Schema;
import io.swagger.v3.oas.annotations.responses.ApiResponse;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

@Controller
@RequestMapping("/api/v1/poc/")
public class POC {

  @Operation(summary = "Get thing",
          responses = {
                  @ApiResponse(description = "Successful Operation", responseCode = "200",
                          content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))),
                  @ApiResponse(responseCode = "404", description = "Not found"),
                  @ApiResponse(responseCode = "401", description = "Authentication Failure")
          }
  )
  @RequestMapping(path = "/xxx", method = RequestMethod.GET)
  ResponseEntity<String> test() {
    return ResponseEntity.ok("Hello");
  }
}
```

1.1.33 Renders like so
![image](https://user-images.githubusercontent.com/4494301/64415603-8ad9b380-d08d-11e9-93b3-8adc09c70900.png)

and 1.1.34-1.1.36 renders
![image](https://user-images.githubusercontent.com/4494301/64415505-5ebe3280-d08d-11e9-9054-076fa7c6703f.png)

**Comment by springdoc** [comment]
Hi, 

 Issue reproduced. It will be possible to handle as return an empty content as response using, one of the following syntax:

```java
@Operation(summary = "Get thing", responses = {
		@ApiResponse(description = "Successful Operation", responseCode = "200", content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))),
		@ApiResponse(responseCode = "404", description = "Not found", content = @Content),
		@ApiResponse(responseCode = "401", description = "Authentication Failure", content = @Content(schema = @Schema(hidden = true))) })
@RequestMapping(path = "/testme", method = RequestMethod.GET)
ResponseEntity<String> testme() {
	return ResponseEntity.ok("Hello");
}
```
 
The fix will be available on v1.1.37 of springdoc-openapi.

**Comment by Stexxen** [comment]
Thanks, confirmed putting `content = @Content` now resolves it
